mkdocs.yml

```diff
@@ -25,7 +25,7 @@ markdown_extensions:
 
 plugins:
   - search
-  - git-revision-date-localized
+#   - git-revision-date-localized
```

Comment:
Why such change? If this should be deleted, it should not be left as a comment.

Reply:
This file should not contain such changes, so either please remove that commented line or revert this change.